core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -38,10 +38,10 @@
 #endif /* ARM64 */
 
 /* PL011 UART */
-#if defined(CFG_HIKEY_UART3)
-#define CONSOLE_UART_BASE	0xF7113000
-#else
+#if defined(CFG_HIKEY_UART0)
```

Comment:
That's a good suggestion Jerome, but I think we need to have two variants available, since we sometime just a single UART but also uses two UARTs in some cases. So I think we need one for normal- and one for secure world. What about `CFG_NW_CONSOLE_UART` and `CFG_SW_CONSOLE_UART`?